File: LightningConstructs/Source/Governor/Declarations.vb

    Public Partial Class Governor
        Private BaseRate As System.Double
        Private BaseTimeConstant As System.Int64
        Private BaseDelta As System.Int64
        Private ReadOnly BaseGovernorWatch As System.Diagnostics.Stopwatch
        Private ReadOnly BaseSleepOffsetConstant As System.Int64
        Private BaseSleepTarget As System.TimeSpan
    End Class

File: LightningConstructs/Source/Governor/Declarations.cs
namespace Lightning
{
    public partial class Governor
    {
        private System.Double BaseRate;
        private System.Int64 BaseTimeConstant;
        private System.Int64 BaseDelta;

        private readonly System.Diagnostics.Stopwatch BaseGovernorWatch;
        private readonly System.Int64 BaseSleepOffsetConstant;
        private System.TimeSpan BaseSleepTarget;
    }
}